### ISSUE TITLE
Project info not appeared

### ISSUE THREAD

**Issue by olgadanylova** [CocoaPods/CocoaPods#9134]
* [X ] I've read and understood the [*CONTRIBUTING* guidelines and have done my best effort to follow](https://github.com/CocoaPods/CocoaPods/blob/master/CONTRIBUTING.md).

# Report

## What did you do?

Pushed my repo to Cocoapods

## What did you expect to happen?

Project's info is appeared on the right side

## What happened instead?

Project's info not appeared for a while

### Stack

```
   CocoaPods : 1.7.4
        Ruby : ruby 2.3.7p456 (2018-03-28 revision 63024) [universal.x86_64-darwin18]
    RubyGems : 2.5.2.3
        Host : Mac OS X 10.14.6 (18G95)
       Xcode : 10.3 (10G8)
         Git : git version 2.20.1 (Apple Git-117)
Ruby lib dir : /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib
Repositories : master - https://github.com/CocoaPods/Specs.git @ b5f80aa9c8e6f815731e9b8d1a390df700e995e5
```

### Installation Source

```
Executable Path: /usr/local/bin/pod
```

## Project that demonstrates the issue

I don't have a project but I'll be glad if info appeared here: 
https://monosnap.com/file/dtHfjaeoRFOZQbnFqYdI05damMsJgL
Everything is ok with another our projects, but this one is the newest.

**Comment by olgadanylova** [comment]
Hello @dnkoutso,
please check this screenshot https://monosnap.com/file/dtHfjaeoRFOZQbnFqYdI05damMsJgL - that's what I mean by the right side.

**Comment by ChrisMarshallNY** [comment]
I am having the exact same issue. I spent yesterday going through my podspec and project, and can't find anything that explains the issue. I am enclosing two screenshots.

One is for a pod that has been on CP for a couple of years (BMLTiOSLib), and the other is for one that I have only recently added (RVS_Spinner).

RVS_Spinner has all the same stuff as BMLTiOSLib, but the project info (and the CHANGELIST) do not appear.
![RVS_Spinner](https://user-images.githubusercontent.com/51290189/64246796-7bb30400-cedb-11e9-8e29-082e658d22ec.png)
![BMLTiOSLibCocoaPod](https://user-images.githubusercontent.com/51290189/64246797-7c4b9a80-cedb-11e9-9027-1d0ded249976.png)

**Comment by olgadanylova** [comment]
@ChrisMarshallNY thanks for sharing!
@dnkoutso any ideas?

**Comment by ChrisMarshallNY** [comment]
Nothing, so far, eh? Not the end of the world...

I should mention that, some time ago, I reported an error, where the project info kept reporting an MIT license as "unknown," even though it was specified in the podspec.

It was never acknowledged, but after some time, I figured out that the parser ignores the podspec if it finds a LICENSE file in the repo, and that I had an extra line at the top of my file that broke the parser.

It's been a while, but I'm pretty sure that I just moved the offending line to the bottom of the file, and it worked. I probably closed the report.

It's entirely possible that I have some kind of error like that in my repo, but I can't figure out what it is.
